docs/getting-started-guides/kubeadm.md

```diff
@@ -78,13 +80,19 @@ Note: `setenforce 0` will no longer be necessary on CentOS once [#33555](https:/
 The master is the machine where the "control plane" components run, including `etcd` (the cluster database) and the API server (which the `kubectl` CLI communicates with).
 All of these components run in pods started by `kubelet`.
 
+Right now you can't run `kubeadm init` twice without turning down the cluster in between, see [Turndown](#turndown).
+
 To initialize the master, pick one of the machines you previously installed `kubelet` and `kubeadm` on, and run:
 
      # kubeadm init
 
 **Note:** this will autodetect the network interface to advertise the master on as the interface with the default gateway.
 If you want to use a different interface, specify `--api-advertise-addresses=<ip-address>` argument to `kubeadm init`.
 
+If you want to use [flannel](https://github.com/coreos/flannel) as the pod network; specify `--pod-network-cidr=10.244.0.0/16` if you're using the daemonset manifest below.
```

Comment:
Ok, may be we should make it more explicit by adding: _However, this is not required for any other networks, including Weave, which is currently recommended below._